apps/design/frontend/src/geography_screen.tsx

```diff
@@ -569,6 +579,52 @@ function PrecinctForm({
                       }
                     />
 
+                    {features.PRECINCT_SPLIT_ELECTION_TITLE_OVERRIDE && (
+                      <InputGroup label="Election Title Override">
+                        <input
+                          type="text"
+                          value={split.electionTitleOverride ?? ''}
+                          onChange={(e) =>
+                            setSplit(index, {
+                              ...split,
+                              electionTitleOverride: e.target.value,
+                            })
+                          }
+                        />
+                      </InputGroup>
+                    )}
+
+                    {features.PRECINCT_SPLIT_CLERK_SIGNATURE_IMAGE && (
+                      <div>
+                        <FieldName>Clerk Signature Image</FieldName>
```

Comment:
For the school district case, I'm wondering if this isn't actually a clerk signature but a school district member signature. (I checked last year's ballots, and we actually didn't have any signatures on the school district ballots, so I'm unsure what's typical.)

We could call these field "Signature Image " and "Signature Caption" to keep things more general. Also open to feedback from Ginny and Tabitha.

Not blocking of course!

Reply:
[ecc2e84](https://github.com/votingworks/vxsuite/pull/5823/commits/ecc2e8407998d722b53e5c0532d081e90c53e631)